game/game.js

```diff
@@ -401,7 +404,16 @@ function canPlay(baseGame, username){
     }else{
         //Medium
         let medium = baseGame.mediums.find(medium => medium.username == username);
-        return medium.hasPlayed ? false : (baseGame.ghost.mediumsHasCards.find(username => username == medium.username) != undefined);
+        
+        if(medium.hasPlayed)
+            return false
+
+        if(baseGame.ghost.mediumsHasCards.find(username => username == medium.username) != undefined)
```

Comment:
What's `baseGame`? What does `base` mean? If it doesn't mean anything, just call it `game`.

Reply:
`mediumsHasCards` has a verb in it, but it's not a function, there's probably a naming problem

Reply:
To check if something is not `undefined` you can do:
```
if(!something) {
     //...
}
```

Reply:
Please read: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness